schema/mysql/schema.sql

```diff
@@ -573,8 +573,8 @@ CREATE TABLE downtime (
   id binary(20) NOT NULL COMMENT 'sha1(environment.name + name)',
   environment_id binary(20) NOT NULL COMMENT 'environment.id',
 
-  triggered_by_id binary(20) DEFAULT NULL COMMENT 'downtime.id',
-  parent_id binary(20) DEFAULT NULL COMMENT 'downtime.id',
+  triggered_by_id binary(20) DEFAULT NULL COMMENT 'The ID of the downtime that triggered this downtime. This is set when creating downtimes on a host higher up in the dependency chain using the "child_option" "DowntimeTriggeredChildren" and can also be set manually via the API.',
```

Comment:
Icinga Web does not provide these options for service downtimes but via the API it's possible, right? I would then write **on a host or service**.